content/misc/products-with-variants.md

```diff
@@ -0,0 +1,187 @@
+ - You have less than 5 axes in your variant group,
+ - The products inside your variant group use the same family,
+ - The products inside your variant group use the same attributes.
+
+It's very good news for you! The solution is to use the new product model and family variant entities, that were introduced in the remodeling of the products with variants for the 2.0.
+
+### A few words about the new modeling
+
+First, let me explain how this new modeling works.
+
+In 2.0, you now can have families that have variants. These variants are called `family variant` and they are used to distribute the family attributes in the different levels of enrichment of your family.
+
+In your case, you will create a family variant with two levels of enrichment, which means you will have one level of variant. You can then choose in the other attributes of the family, which one will be your common attributes and which one will differ depending of the variant products.
+
+The common attributes of your product will be stored in a new entity called `product model`. It's an entity which is really similar to classical products as its aim is to be the receptacle of the common attribute values of your variant products.
+
+All the other attributes have their value stored into standard products that we call `variant product`.
+
+To be much clearer, below is a diagram representing this modeling.
+
+![Modeling of the products with variants](../img/product_with_variants.png)
+
+### Example  
+In the API, you will end up with the following change in your products with variants.   
+Before in 1.7
+```json
+{
+  "identifier": "1111111195",
+  "family": "clothing",
+  "groups": [],
+  "variant_group": "my_structured_variant_group",
+  "categories": [ "tshirts"],
+  "enabled": true,
+  "values": {},
+  "created": "2017-10-05T11:25:48+02:00",
+  "updated": "2017-10-05T11:25:48+02:00",
+  "associations": {},
+  "metadata": {
+    "workflow_status": "working_copy"
+  }
+}
+```
+Now in 2.0
+```json
+{
+  "identifier": "1111111195",
+  "family": "clothing",
+  "groups": [],
+  "parent": "my_product_model",
+  "categories": ["tshirts"],
+  "enabled": true,
+  "values": {},
+  "created": "2017-10-05T11:25:48+02:00",
+  "updated": "2017-10-05T11:25:48+02:00",
+  "associations": {},
+  "metadata": {
+    "workflow_status": "working_copy"
+  }
+}
+```
+
+::: info
+For this solution to work, do not forget to create a product model with the code `my_product_model`. The family `clothing` used here, should have a family variants that is linked to this product model. ;)
+:::
+
+## My variant groups are a real mess
+
+Your variant groups are not structured enough:
+ - You have more than 5 axes in your variant group,
+ - The products inside your variant group does not use the same family,
+ - The products inside your variant group does not use the same attributes.
+
+You will only have one choice here: think about a better modeling for these products.
```

Comment:
:confident:

Reply:
That's the rule Delphine gave me. We have to push our customers toward better catalog modeling. A catalog without a structure is a bad an inefficient catalog. :)